hack/verify-starlark.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+VERSION="0.29.0"
+
+MODE="check"
+
+if [[ "$*" == "fix" ]]; then
+  MODE="fix"
+fi
+
+if [[ "${OSTYPE}" == "linux"* ]]; then
+  BINARY="buildifier"
+elif [[ "${OSTYPE}" == "darwin"* ]]; then
+  BINARY="buildifier.mac"
+fi
+
+# shellcheck source=./hack/utils.sh
+source "$(dirname "$0")/utils.sh"
+ROOT_PATH=$(get_root_path)
+
+# create a temporary directory
+TMP_DIR=$(mktemp -d)
+
+# cleanup on exit
+cleanup() {
+  ret=0
+  if [[ -s "${OUT}" ]]; then
+    echo "Found errors:"
+    cat "${OUT}"
+    ret=1
+  fi
+  echo "Cleaning up..."
+  rm -rf "${TMP_DIR}"
+  exit ${ret}
+}
+trap cleanup EXIT
+
+# install buildifier
+cd "${TMP_DIR}" || exit
+
+curl -L "https://github.com/bazelbuild/buildtools/releases/download/${VERSION}/${BINARY}" -o buildifier
```

Comment:
Can we put this in one of the gitignored directories and only download it if it doesn't already exist? Could use hack/tools/bin perhaps?

Reply:
Done. Also for shellcheck.